components/MainNavBar.vue

```diff
@@ -101,7 +99,7 @@ export default {
             { name: "Timeline Search", url: "/", external: false },
             { name: "Residences on ArcGIS Online", url: "/", external: false },
             {
-              name: "Residences Within Each Parish",
+              name: "Residences Withgit in Each Parish",
```

Comment:
Looks like a typo has been introduced here.

Reply:
done

Reply:
Sorry Ruby it still isn't quite right. It says _within in_

Reply:
oh silly me done now